### PR TITLE
jax.Array tutorial

### PR DESCRIPTION
**Pull request by yashk2810**

### CODE REVIEW

File: docs/jax_Array.md [hunk 1]
```diff
@@ -0,0 +1,854 @@
...
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.1
+kernelspec:
+  display_name: Python 3
+  name: python3
+---
+
```
Comment (jakevdp) [inline docs/jax_Array.md:13]: Add a top-level heading?

**Review by jakevdp — COMMENTED** [review]
Looks good - we'll need to add this to a table of contents somewhere in order for it to be rendered on the website. Maybe under advanced JAX tutorials?

File: docs/jax_Array.md (hunk 1 above)
Comment (yashk2810) [reply docs/jax_Array.md:13]: Done

**Comment by yashk2810** [comment]
Done. Added it to advanced tutorial section!

File: docs/notebooks/jax_Array.md [hunk 2]
```diff
@@ -0,0 +1,858 @@
...
+    format_version: 0.13
+    jupytext_version: 1.14.1
+kernelspec:
+  display_name: Python 3
+  name: python3
+---
+
++++ {"id": "PxHrg4Cjuapm"}
+
+# `jax.Array`
```
Comment (jakevdp) [inline docs/notebooks/jax_Array.md:16]: This title is what will be rendered in the TOC. Can we use something more descriptive?

**Comment by jakevdp** [comment]
Also, there are a few errors in the readthedocs build that should be addressed here.

File: docs/notebooks/jax_Array.md [hunk 3]
```diff
@@ -0,0 +1,858 @@
...
+
++++ {"id": "pFtQjv4SzHRj"}
+
+**The features required by `jax.Array` are not supported by the Colab TPU runtime at this time.**
```
Comment (jakevdp) [inline docs/notebooks/jax_Array.md:20]: I might add a note here to orient the reader, something like:
```
This tutorial discusses parallelism via `jax.Array`, the unified array object model available in JAX vX.X.X and newer.
```

File: docs/notebooks/jax_Array.md (hunk 2 above)
Comment (yashk2810) [reply docs/notebooks/jax_Array.md]: Done

File: docs/notebooks/jax_Array.md (hunk 3 above)
Comment (yashk2810) [reply docs/notebooks/jax_Array.md:22]: Done.

**Comment by jakevdp** [comment]
It looks like notebook execution failed... if you want to avoid doctest execution you can add it to the list here: https://github.com/google/jax/blob/726b2bc2ee1bbc5690ad2b6ab0e2b77aa88eb868/docs/conf.py#L111

**Review by jakevdp — APPROVED** [review]
Looks great!

**Pull request merged**